modules/auxiliary/admin/http/mantisbt_password_reset.rb

```diff
@@ -0,0 +1,109 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+## Current source: https://github.com/rapid7/metasploit-framework
+###
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'       => "MantisBT password reset",
+      'Description'  => %q{
+        MantisBT before 1.3.10, 2.2.4, and 2.3.1 are vulnerable to unauthenticated password reset.
+      },
+      'License'    => MSF_LICENSE,
+      'Author'     =>
+        [
+          'John (hyp3rlinx) Page',  # initial discovery
+          'Julien (jvoisin) Voisin' # metasploit module
+        ],
+      'References'   =>
+        [
+          ['CVE', '2017-7615'],
+          ['EDB', '41890'],
+          ['URL', 'https://mantisbt.org/bugs/view.php?id=22690']
+                    ['URL', 'http://hyp3rlinx.altervista.org/advisories/MANTIS-BUG-TRACKER-PRE-AUTH-REMOTE-PASSWORD-RESET.txt']
+        ],
+      'Platform'     => ['win', 'linux'],
+      'Privileged'   => false,
+      'DisclosureDate' => "Apr 16, 2017"))
```

Comment:
I don't think `,` is allowed in `DisclosureDate`.

Make sure you run `tools/dev/msftidy.rb`